database/mongoDb.js

```diff
@@ -39,47 +39,31 @@ let Rating = mongoose.model('Rating', ratingSchema);
 const getAllReviews = () => {
   return new Promise ((resolve, reject) => {
     Review.find((err, reviews) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(reviews);
-      }
+      err ? reject(err) : resolve(reviews);
     });
   });
 };
 
 const getReviewsForOneCourse = (id) => {
   return new Promise ((resolve, reject) => {
     Review.find({courseId: id}, (err, reviews) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(reviews);
-      }
+      err ? reject(err) : resolve(reviews);
     });
   });
 };
 
 const getAllRatings = () => {
   return new Promise ((resolve, reject) => {
     Rating.find((err, ratings) => {
-      if (err) {
-        reject(err);
-      } else {
-        resolve(ratings);
-      }
+      err ? reject(err) : resolve(ratings);
     });
   });
 };
 
 const getRatingForOneCourse = (id) => {
```

Comment:
This function is a bit big. Some of this could be refactored to a controller function